tools/output/vehrouteDiff.py

```diff
@@ -25,14 +25,26 @@
 sys.path.append(os.path.join(os.path.dirname(sys.argv[0]), '..'))
 from sumolib.output import parse  # noqa
 from sumolib.miscutils import uMax, Statistics, parseTime  # noqa
+from sumolib.options import ArgumentParser 
 
 
+def parse_args():
+    USAGE = "Usage: " + sys.argv[0] + " <orig> <new> <out> [--earliest]"
+    optParser = ArgumentParser()
+    optParser.add_argument("orig", help="Original XML File")
+    optParser.add_argument("new", help="New XML File")
+    optParser.add_argument("out", help = "Output File")
+    optParser.add_option("--earliest", action="store_true",
+                         default=None, help="Earliest Output Parameter (Default: None)")
```

Comment:
the default should be False here

Reply:
Updated here: Commit 68a8cd4be1837f0a1d27d853da6ccabed4628210